citi-onboarding/Maracatec
# (feature/Front-Carousel): The front of Carousel is done, but i have to connect with the back yet

What i did 
----------
I've done the components of the carousel as well the function and settings that make him work perfectly. 


How to Test
----------

1. Open the terminal and go to the folder of this project

2. In terminal:

  i. `git checkout develop`
  ii. `git pull`
  iii. `git checkout feature/Timer`
  iv. `git pull origin feature/Timer`
  v. `open the folder server with comand cd client/`
  vi. `yarn install or npm install`
  vii. `yarn start or npm start`

3. Check if the component looks just like the mockup in different screen sizes (+1200px)

4. Read the changed files section on GitHub

5. Test the things you think are worth testing, even the ones that are not in this description
